### DISCUSSION TITLE
Getting the hands to animate on basic model

### DISCUSSION THREAD

**Discussion by hollyhudson** [mozilla/hubs#3491] [Avatar Questions] [answered]
I've got hands that don't respond to VR controller presses.  They just stay in the flat configuration, like I'm waving spatulas around.

I grabbed a clean copy of the AvatarBot_base_for_export.blend file, I edited the materials to give it a T-shirt and jeans (so I didn't touch the mesh at all).  I added a scale with audio feedback modifier on the head bone, and I added a loop animation component for idle_eyes.

In the export menu I unselected "Sample all animations", I selected "Apply modifiers", the avatar root and the mesh object are both selected and the "Selected Objects" item is clicked in the export window, and the Hubs Components selection is also clicked in the export window.

Those are the only things I changed.

Most of the avatar works -- it loads in Hubs, idle-eyes plays (haven't checked if the head scaling works yet), but the hands, they are dead.  What am I missing?  

[Here is my blend file.](https://github.com/hackerspace-zone/zone-scene/blob/main/resistor-T-skirt.blend)
[And here is the glb file.](https://github.com/hackerspace-zone/zone-scene/blob/main/resistor-T-skirt.glb)

**Comment by j-conrad** [comment, marked as answer]
Looking at your .glb in a simple glTF viewer like this one:
https://gltf-viewer.donmccurdy.com/

You'll see that there are no other animations other than "idle_eyes". That's usually the first clue.

What I found in your .blend file was a simple, but somewhat annoying quirk. If you go to the Animation workspace, you'll see that you have the 'idle_eyes' animation selected and in 'edit' mode (TAB). If an animation is considered Active like this, then it doesn't export correctly. All I had to do was hit TAB again to turn OFF that mode (green goes away), then re-export. 

![image](https://user-images.githubusercontent.com/31968496/101218105-51e51180-3637-11eb-9610-bf06debf30db.png)

Testing in the same viewer afterward shows all the animations listed:
![image](https://user-images.githubusercontent.com/31968496/101218159-69bc9580-3637-11eb-98e4-0092bdbe9945.png)

**Comment by hollyhudson** [comment]
That worked!  Perfect!  Thanks so much!
